### PR TITLE
Added Elastic Stack

### PR DESCRIPTION
**Pull request by MadelineMurray**
Signed-off-by: Madeline Murray <madeline.taylor@gmail.com>

Added Elastic Stack

### CODE REVIEW

File: docs/HowTo/Monitor/Elastic-Stack.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# Elastic Stack 
+
+Elastic Stack (also known as ELK) is a common open-source log management platform. 
+
+The [Besu Quickstart](https://github.com/PegaSysEng/besu-quickstart) provides an example implementation 
+using Elastic Stack for log management. 
+
+The [Filebeat configuration](https://github.com/PegaSysEng/besu-quickstart/blob/master/filebeat/filebeat.yml)
+ingests the logs and outputs them to Redis for storage. Using Redis enables logs to be stored and pulled 
+by Logstash as needed. Using Redis provides a highly available log management configuration.
```
Comment (joshuafernandes) [inline docs/HowTo/Monitor/Elastic-Stack.md]: Suggest the last 2 lines read something like this `Redis provides a highly available mechanism which enables logs to be stored by any of the Elastic Beats and pulled by Logstash as required.`

File: docs/HowTo/Monitor/Elastic-Stack.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+!!! note
+    The pipeline configuration must match the your log format. If using the default, you can use the 
+    [grok plugin](https://www.elastic.co/guide/en/logstash/current/plugins-filters-grok.html) to 
+    extract the log fields. 
+
+To see the Besu Quickstart logs in Kibana:  
+
+1. [Start the Besu Quickstart](../../Tutorials/Quickstarts/Private-Network-Quickstart.md).
+1. Open the [`Kibana logs address`](http://localhost:5601/app/kibana#/discover) listed by the Besu Quickstart.
```
Comment (joshuafernandes) [inline docs/HowTo/Monitor/Elastic-Stack.md]: suggest "Kibana logs visualisation address"?

File: docs/HowTo/Monitor/Elastic-Stack.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/HowTo/Monitor/Elastic-Stack.md]: Updated

File: docs/HowTo/Monitor/Elastic-Stack.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/HowTo/Monitor/Elastic-Stack.md]: This matches what's listed by the quickstart so I think I'll leave it matching

**Pull request merged**